web-app/src/utilities/AxiosInstance.js

```diff
@@ -37,6 +38,24 @@ instance.interceptors.response.use(function(response) {
     };
   }
 
+  // Append employee to each event (we know its the logged in user) and convert
+  // dates to moment objects
+  if (response.config.url.includes(`${baseURL}/dashboard/getEmployeeEvents`)) {
+    const events = [...response.data.events];
+    const employee = store.getState().USER;
+    for (const index in events) {
```

Comment:
small suggestion but I would use `property` as opposed to `index` here - as I believe events is an object as opposed to an array? this will make the code a bit more self documenting and understandable.

If event events is an array as opposed to an object you should use a for OF loop

Reply:
It's an array of objects. Changed.